CHANGELOG.md

```diff
@@ -14,6 +15,130 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+## v0.5.0-rc1 (unreleased)
+
+API versions: v1beta1, v1alpha2
+
+This release is all about stability.
+
+Changes in this release can largely be divided into the following categories:
+
+- Resources graduating to beta
```

Comment:
I suggest we add a "Release channels" item here.

Reply:
Added in 0b7da46768f257130ba5d63dbad89314b23e8dd0 :+1: